battery.sh

```diff
@@ -477,6 +487,11 @@ if [[ "$action" == "maintain" ]]; then
 		kill $pid &>/dev/null
 	fi
 
+	if test -f "$calibrate_pidfile"; then
+		pid=$(cat "$calibrate_pidfile" 2>/dev/null)
+		log "🚨 Calibration is running, please run 'battery calibrate stop' or wait for it to finish"
```

Comment:
Same as above, the logic should exit here so clashes do not just cause error logs. During a calibration there should either:

1. Be a blocking of maintenance commands
2. The maintenance commands should stop the calibration and give a verbose output of "hey I stopped the calibration"

Reply:
Also here? Should stop the calibration process?